Tests/OpenAPIRuntimeTests/URICoder/Parsing/Test_URIParser.swift

```diff
@@ -62,7 +65,8 @@ final class Test_URIParser: Test_Runtime {
                     simpleExplode: .custom("Hello%20World", value: ["": ["Hello World"]]),
                     simpleUnexplode: .custom("Hello%20World", value: ["": ["Hello World"]]),
                     formDataExplode: "hello=Hello+World",
-                    formDataUnexplode: "hello=Hello+World"
+                    formDataUnexplode: "hello=Hello+World",
+                    deepObjectExplode: "hello=Hello%20World"
```

Comment:
Shouldn't this have some square brackets?

Reply:
You mean the correct output should be `[hello]=Hello%20World` ?